### PR TITLE
Add draftstore

### PR DESCRIPTION
**Pull request by naikajah**
**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: k8s/perftest/cluster-00/static/neuvector/neuvector.yaml [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+        secretName: storage-secret
+        shareName: neuvector-data-00
+      configmap:
+        enabled: true
+        data:
+          samlinitcfg.yaml: |
+            SSO_URL: https://login.microsoftonline.com/531ff96d-0ae9-462a-8d2d-bec7c0b42082/saml2
+            Issuer: https://sts.windows.net/531ff96d-0ae9-462a-8d2d-bec7c0b42082/
+            X509_Cert: |
+              -----BEGIN CERTIFICATE-----
```
Comment (timja) [inline k8s/perftest/cluster-00/static/neuvector/neuvector.yaml]: Certs are different per environment this is the guide that devops follow normally:
https://tools.hmcts.net/confluence/display/CNP/Neuvector+AAD+integration

File: k8s/perftest/cluster-01/static/neuvector/neuvector.yaml [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+  annotations:
+    flux.weave.works/ignore: "false"
+spec:
+  releaseName: neuvector
+  chart:
+    git: https://github.com/hmcts/neuvector-helm
+    ref: config-map
+    path: "."
+  values:
+    registry: hmctsprivate.azurecr.io
```
Comment (timja) [inline k8s/perftest/cluster-01/static/neuvector/neuvector.yaml]: shouldn't need any of this config if you use the kustomise base? see the aat neuvector file

File: k8s/perftest/common/draft-store/draft-store-service.yaml [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+    flux.weave.works/automated: "true"
+    flux.weave.works/tag.java: glob:prod-*
+spec:
+  releaseName: draft-store-service
+  chart:
+    repository: https://hmcts.azurecr.io/helm/v1/repo/
+    name: draft-store-service
+    version: 1.0.3
+  values:
+    java:
```
Comment (timja) [inline k8s/perftest/common/draft-store/draft-store-service.yaml]: shouldn't this be set @adusumillipraveen ?
`useInterpodAntiAffinity: true`

and all the pdb flags?
https://github.com/hmcts/chart-java

File: k8s/perftest/common/pod-identity/pod-identity.yaml [hunk 4]
```diff
@@ -0,0 +1,205 @@
+apiVersion: v1
```
Comment (timja) [inline k8s/perftest/common/pod-identity/pod-identity.yaml]: I think we could just move this to common and bin the old AAT? 
@adusumillipraveen @luigibk ?

Comment (adusumillipraveen) [reply k8s/perftest/common/pod-identity/pod-identity.yaml]: Yes, agree for this.

File: k8s/perftest/cluster-00/static/neuvector/neuvector.yaml (hunk 1 above)
Comment (naikajah) [reply k8s/perftest/cluster-00/static/neuvector/neuvector.yaml]: so based on the below comment we don't require this.

Comment (timja) [reply k8s/perftest/cluster-00/static/neuvector/neuvector.yaml]: we do require the config map config just not the version of neuvector / where to get the chart from

File: k8s/perftest/common/draft-store/draft-store-service.yaml (hunk 3 above)
Comment (adusumillipraveen) [reply k8s/perftest/common/draft-store/draft-store-service.yaml]: Yes, AAT is better to refer : https://github.com/hmcts/cnp-flux-config/blob/master/k8s/aat/common/rpe/draft-store/draft-store-service.yaml

File: k8s/perftest/common/pod-identity/pod-identity.yaml (hunk 4 above)
Comment (naikajah) [reply k8s/perftest/common/pod-identity/pod-identity.yaml]: moved the pod identity to common

File: k8s/perftest/common/draft-store/draft-store-service.yaml (hunk 3 above)
Comment (timja) [reply k8s/perftest/common/draft-store/draft-store-service.yaml]: even AAT is missing pdb 👅

Comment (adusumillipraveen) [reply k8s/perftest/common/draft-store/draft-store-service.yaml]: PDB is set to 50% by default.

Comment (timja) [reply k8s/perftest/common/draft-store/draft-store-service.yaml]: ah I missed that

Comment (naikajah) [reply k8s/perftest/common/draft-store/draft-store-service.yaml]: done..

File: k8s/perftest/common/draft-store/draft-store-service.yaml [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+metadata:
+  name: draft-store-service
+  namespace: rpe
+  annotations:
+    flux.weave.works/automated: "true"
+    flux.weave.works/tag.java: glob:prod-*
+spec:
+  releaseName: draft-store-service
+  chart:
+    repository: https://hmcts.azurecr.io/helm/v1/repo/
```
Comment (timja) [inline k8s/perftest/common/draft-store/draft-store-service.yaml]: this is the wrong helm chart repo.
```suggestion
    repository: https://hmctspublic.azurecr.io/helm/v1/repo/
```

File: k8s/perftest/common/draft-store/draft-store-service.yaml [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+    name: draft-store-service
+    version: 1.0.4
+  values:
+    rollback:
+      enable: true
+    java:
+      replicas: 2
+      useInterpodAntiAffinity: true
+      applicationInsightsInstrumentKey: "88e9f6bc-d5ca-4bae-b408-03c271caa4a0"
```
Comment (timja) [inline k8s/perftest/common/draft-store/draft-store-service.yaml]: @adusumillipraveen should we move this to vault instead? or not worth it?  can't remember where we got to with this

File: k8s/perftest/common/draft-store/draft-store-service.yaml [hunk 7]
```diff
@@ -0,0 +1,45 @@
...
+        SPRING_APPLICATION_NAME: "HMCTS Draft Store AKS"
+      aadIdentityName: rpe
+      keyVaults:
+        "draft-store":
+          resourceGroup: draft-store-service
+          secrets:
+          - service-POSTGRES-PASS
+    global:
+      environment: perftest
+      subscriptionId: "8a07fdcd-6abd-48b3-ad88-ff737a4b9e3c"
```
Comment (timja) [inline k8s/perftest/common/draft-store/draft-store-service.yaml]: this should be for where the app team infra is:
```suggestion
      subscriptionId: "7a4e3bd5-ae3a-4d0c-b441-2188fee3ff1c"
```

File: k8s/perftest/common/draft-store/draft-store-service.yaml (hunk 6 above)
Comment (adusumillipraveen) [reply k8s/perftest/common/draft-store/draft-store-service.yaml]: I think its worth it to move now, we will be setting up a bad example to others i guess.

**Pull request merged**